ControlRoomApplication/ControlRoomApplication/Database/Operations/DatabaseOperations.cs

```diff
@@ -906,6 +906,45 @@ public static RadioTelescope FetchRadioTelescopeByID(int id)
             
         }
 
+        /// <summary>
+        /// Adds a selected weather Threshold to the database
+        /// </summary>
+        /// <param name="weatherThreshold"></param>
+        public static void AddWeatherThreshold(WeatherThreshold weatherThreshold)
+        {
+            using (RTDbContext Context = InitializeDatabaseContext())
+            {
+                Context.WeatherThreshold.Add(weatherThreshold);
+                SaveContext(Context);
+
+                logger.Info(Utilities.GetTimeStamp() + ": Added WeatherThreshold to database");
+            }
+        }
+
+        /// <summary>
+        /// Routine to retrieve the time interval for dumping snow off of the dish (in minutes) from the database
+        /// </summary>
+        public static int FetchSnowDumpTime()
```

Comment:
I think instead of making this `FetchSnowDumpTime`, it would be good for it to fetch the whole `WeatherThreshold`, because we will likely need to use the wind speed somewhere too (and then this function would take care of it).